cycledash/static/js/examine/RecordStore.js

```diff
@@ -379,7 +379,7 @@ function createRecordStore(run, igvHttpfsUrl, dispatcher, opt_testDataSource) {
     var search = window.location.search.substring(1),
         vars = search.split('&');
     var val = _.first(_.filter(vars, v => {
-      var [key, val] = v.split('=');
+      var [key] = v.split('=');
```

Comment:
I'm not sure what the best choice is here; in other languages leaving out the second var to be unpacked would result in an error. Maybe the best choice is to turn off this lint rule? Could also do 

```javascript
var key = v.split('=')[0];
```